### PR TITLE
Localization/drop major 01 08 2020

### PR DESCRIPTION
**Pull request by jballance**

### CODE REVIEW

**Comment by jballance** [comment]
@theeagle it looks like the strings imported fine after moving them up a directory. (I did a full export/re-import)

File: de.lproj/ShortcutRecorder.strings [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/* No comment provided by engineer. */
+"Click to record shortcut" = "Hier klicken, um Tastenkombination aufzuzeichnen";
+
+/* No comment provided by engineer. */
+"Command-" = "%DE string testing%-";
```
Comment (jballance) [inline de.lproj/ShortcutRecorder.strings:5]: I changed this translation manually and then pushed it to git to test if it worked, and it looks like it did

Comment (theeagle) [reply de.lproj/ShortcutRecorder.strings:5]: awesome

**Comment by theeagle** [comment]
Can you update the encodings to utf-8? I did that originally, but then backed out those changes.

**Comment by jballance** [comment]
sure, I just pushed that up to #19

**Comment by theeagle** [comment]
I'm getting this error: Bildschirmfoto 2020-01-10 um 10 41 28 AM

**Comment by jballance** [comment]
hmmm. when I changed the encoding it asked me if I wanted to convert the files and I thought I clicked yes, but maybe not?

**Comment by theeagle** [comment]
well, this is on the branch that didn't have the encodings changed, right?

**Comment by theeagle** [comment]
it acted like the files themselves were utf-8, but the project settings for the file (on the right) showed utf-16

**Comment by jballance** [comment]
ohh I'm going to guess that in the import process, the new strings files we wrote out were utf8, but I didn't update the project settings in this branch

**Comment by jballance** [comment]
since the importer just writes out the new files and overwrites

**Comment by jballance** [comment]
I'm starting another import, we can see if this one works better since hopfully project settings should expect utf8, and the .strings should be utf8

**Pull request closed without merging**